server/src/test/java/io/unitycatalog/server/base/catalog/BaseCatalogCRUDTest.java

```diff
@@ -53,10 +53,15 @@ public void testCatalogCRUD() throws ApiException {
         CatalogInfo catalogInfo2 = catalogOperations.getCatalog(CATALOG_NAME);
         Assert.assertEquals(catalogInfo, catalogInfo2);
 
-        // Update catalog
-        System.out.println("Testing update catalog..");
-        CatalogInfo updatedCatalogInfo = catalogOperations.updateCatalog(CATALOG_NAME, CATALOG_NEW_NAME, CATALOG_NEW_COMMENT);
-        assertCatalog(updatedCatalogInfo, CATALOG_NEW_NAME, CATALOG_NEW_COMMENT);
+        // Update catalog name
+        System.out.println("Testing update catalog: changing name..");
+        CatalogInfo updatedCatalogInfo = catalogOperations.updateCatalog(CATALOG_NAME, CATALOG_NEW_NAME, null);
+        assertCatalog(updatedCatalogInfo, CATALOG_NEW_NAME, COMMENT);
+
+        // Update catalog comment
```

Comment:
To make this more clear maybe: `// Update catalog comment without updating name`